numpy/_array_api/__init__.py

```diff
@@ -0,0 +1,165 @@
+"""
+A NumPy sub-namespace that conforms to the Python array API standard.
+
+This is a proof-of-concept namespace that wraps the corresponding NumPy
+functions to give a conforming implementation of the Python array API standard
+(https://data-apis.github.io/array-api/latest/). The standard is currently in
+an RFC phase and comments on it are both welcome and encouraged. Comments
+should be made either at https://github.com/data-apis/array-api or at
+https://github.com/data-apis/consortium-feedback/discussions.
+
+This submodule will be accompanied with a NEP (not yet written) proposing its
+inclusion in NumPy.
+
+NumPy already follows the proposed spec for the most part, so this module
+serves mostly as a thin wrapper around it. However, NumPy also implements a
+lot of behavior that is not included in the spec, so this serves as a
+restricted subset of the API. Only those functions that are part of the spec
+are included in this namespace, and all functions are given with the exact
+signature given in the spec, including the use of position-only arguments, and
+omitting any extra keyword arguments implemented by NumPy but not part of the
+spec. Note that the array object itself is unchanged, as implementing a
+restricted subclass of ndarray seems unnecessarily complex for the purposes of
+this namespace, so the API of array methods and other behaviors of the array
+object will include things that are not part of the spec.
```

Comment:
This is a bit fuzzing: by "array object" here I assume it refers to `numpy.ndarray`, not `numpy._array_api.ndarray`?